pkg/rpm/postupgrade

```diff
@@ -139,16 +139,25 @@ docker run --rm -d --name $SVC_NAME -p 9200:9200 \
   sh -c "$ELASTIC_BIN -f -Des.node.name=$NODE_NAME \
   -Des.cluster.name=$CLUSTER_NAME"
 
-if check_elasticsearch; then
-    report "SUCCESS" "Container started within timeout"
+#
```

Comment:
```cp -ar $HOST_ISVCS_DIR/elasticsearch-serviced $HOST_ISVCS_DIR/elasticsearch-serviced.backup```
ES backup shouldn't run on the delegate host

```
Starting docker container elasticsearch-serviced ...
```
shouldn't run on the delegate host